core/domain/rights_manager.py

```diff
@@ -396,60 +581,155 @@ def _can_delete(self, rights_object):
             is_deleting_own_private_object or is_mod_deleting_public_object)
 
     def is_owner(self, activity_type, activity_id):
+        """Returns whether this user is the owner of this activity.
+
+        Args:
+            activity_type: The type of activity. Either one of these:
+                feconf.ACTIVITY_TYPE_EXPLORATION
+                feconf.ACTIVITY_TYPE_COLLECTION
+            activity_id: str. ID of activity.
+
+        Returns:
+            bool. Whether the user is owner of this activity.
+        """
         activity_rights = _get_activity_rights(activity_type, activity_id)
         if activity_rights is None:
             return False
         return self._is_owner(activity_rights)
 
     def has_editing_rights(self, activity_type, activity_id):
-        """Whether this user has editing rights for this activity.
+        """Returns whether this user has editing rights for given activity.
+
+        Args:
+            activity_type: The type of activity. Either one of these:
+                feconf.ACTIVITY_TYPE_EXPLORATION
+                feconf.ACTIVITY_TYPE_COLLECTION
+            activity_id: str. ID of activity.
 
-        This is true if the activity is community-owned, or if the user is in
-        the owner/editor list for the activity.
+        Returns:
+            bool. Whether the user is in the owner/editor list for the activity.
         """
         activity_rights = _get_activity_rights(activity_type, activity_id)
         if activity_rights is None:
             return False
         return self._has_editing_rights(activity_rights)
 
     def has_viewing_rights(self, activity_type, activity_id):
+        """Returns whether this user has viewing rights for given activity.
+
+        Args:
+            activity_type: The type of activity. Either one of these:
+                feconf.ACTIVITY_TYPE_EXPLORATION
+                feconf.ACTIVITY_TYPE_COLLECTION
+            activity_id: str. ID of activity.
+
+        Returns:
+            bool. Whether the user has a right to view an activity.
```

Comment:
a right --> rights (or "the right")
an --> the given

Ditto below, for a few more similar methods.